<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>Game3Score</class>
 <widget class="QWidget" name="Game3Score">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>1280</width>
    <height>720</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Form</string>
  </property>
  <widget class="QTextBrowser" name="textVices">
   <property name="geometry">
    <rect>
     <x>830</x>
     <y>60</y>
     <width>256</width>
     <height>501</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <family>Asap Condensed</family>
     <pointsize>16</pointsize>
    </font>
   </property>
   <property name="styleSheet">
    <string notr="true">background:transparent;
border:none;
color:white;</string>
   </property>
  </widget>
  <widget class="QGraphicsView" name="graphicsView">
   <property name="geometry">
    <rect>
     <x>90</x>
     <y>80</y>
     <width>50</width>
     <height>50</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">background:transparent;
border:0px;
</string>
   </property>
  </widget>
  <widget class="QLabel" name="label_5">
   <property name="geometry">
    <rect>
     <x>560</x>
     <y>0</y>
     <width>121</width>
     <height>31</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <family>Asap Condensed</family>
     <pointsize>16</pointsize>
     <underline>true</underline>
    </font>
   </property>
   <property name="styleSheet">
    <string notr="true">background:transparent;
color:white;
</string>
   </property>
   <property name="text">
    <string>YOUR VALUES:</string>
   </property>
  </widget>
  <widget class="QLabel" name="response">
   <property name="geometry">
    <rect>
     <x>80</x>
     <y>490</y>
     <width>401</width>
     <height>61</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <family>Asap Condensed</family>
     <pointsize>16</pointsize>
    </font>
   </property>
   <property name="styleSheet">
    <string notr="true">background: transparent;
color:white;
</string>
   </property>
   <property name="text">
    <string/>
   </property>
  </widget>
  <widget class="QLabel" name="scoreValue">
   <property name="geometry">
    <rect>
     <x>240</x>
     <y>360</y>
     <width>161</width>
     <height>100</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <family>Asap Condensed</family>
     <pointsize>44</pointsize>
    </font>
   </property>
   <property name="styleSheet">
    <string notr="true">background:transparent;
color:white;
</string>
   </property>
   <property name="text">
    <string/>
   </property>
  </widget>
  <widget class="QLabel" name="label_4">
   <property name="geometry">
    <rect>
     <x>30</x>
     <y>360</y>
     <width>161</width>
     <height>100</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <family>Asap Condensed</family>
     <pointsize>44</pointsize>
    </font>
   </property>
   <property name="styleSheet">
    <string notr="true">background:transparent;
color:white;
</string>
   </property>
   <property name="text">
    <string>SCORE:</string>
   </property>
  </widget>
  <widget class="QGraphicsView" name="graphicsView_2">
   <property name="geometry">
    <rect>
     <x>150</x>
     <y>80</y>
     <width>50</width>
     <height>50</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">background:transparent;
border:0px;
</string>
   </property>
  </widget>
  <widget class="QLabel" name="label_6">
   <property name="geometry">
    <rect>
     <x>910</x>
     <y>0</y>
     <width>121</width>
     <height>31</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <family>Asap Condensed</family>
     <pointsize>16</pointsize>
     <underline>true</underline>
    </font>
   </property>
   <property name="styleSheet">
    <string notr="true">background:transparent;
color:white;
</string>
   </property>
   <property name="text">
    <string>YOUR VICES:</string>
   </property>
  </widget>
  <widget class="QTextBrowser" name="textValues">
   <property name="geometry">
    <rect>
     <x>490</x>
     <y>60</y>
     <width>256</width>
     <height>501</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <family>Asap Condensed</family>
     <pointsize>16</pointsize>
    </font>
   </property>
   <property name="styleSheet">
    <string notr="true">background:transparent;
border:none;
color:white;</string>
   </property>
   <property name="html">
    <string>&lt;!DOCTYPE HTML PUBLIC &quot;-//W3C//DTD HTML 4.0//EN&quot; &quot;http://www.w3.org/TR/REC-html40/strict.dtd&quot;&gt;
&lt;html&gt;&lt;head&gt;&lt;meta name=&quot;qrichtext&quot; content=&quot;1&quot; /&gt;&lt;style type=&quot;text/css&quot;&gt;
p, li { white-space: pre-wrap; }
&lt;/style&gt;&lt;/head&gt;&lt;body style=&quot; font-family:'Asap Condensed'; font-size:16pt; font-weight:400; font-style:normal;&quot;&gt;
&lt;p style=&quot;-qt-paragraph-type:empty; margin-top:0px; margin-bottom:0px; margin-left:0px; margin-right:0px; -qt-block-indent:0; text-indent:0px;&quot;&gt;&lt;br /&gt;&lt;/p&gt;&lt;/body&gt;&lt;/html&gt;</string>
   </property>
  </widget>
  <widget class="QGraphicsView" name="graphicsView_3">
   <property name="geometry">
    <rect>
     <x>210</x>
     <y>80</y>
     <width>50</width>
     <height>50</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">background:transparent;
border:0px;
</string>
   </property>
  </widget>
  <widget class="QLabel" name="label_2">
   <property name="geometry">
    <rect>
     <x>30</x>
     <y>110</y>
     <width>51</width>
     <height>17</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <family>Asap Condensed</family>
     <pointsize>16</pointsize>
    </font>
   </property>
   <property name="styleSheet">
    <string notr="true">background:transparent;
color:white;
</string>
   </property>
   <property name="text">
    <string>LIVES:</string>
   </property>
  </widget>
  <widget class="QPushButton" name="exitButton">
   <property name="geometry">
    <rect>
     <x>1100</x>
     <y>610</y>
     <width>99</width>
     <height>27</height>
    </rect>
   </property>
   <property name="text">
    <string>EXIT</string>
   </property>
  </widget>
 </widget>
 <resources/>
 <connections/>
</ui>
